### DISCUSSION TITLE
can't enter repository for pages

### DISCUSSION THREAD

**Discussion by Jiazi345** [community/community#37879] [Other Feature Feedback, Questions, & Ideas] [answered]
I creat a  username.github.io repository, It is public and I've check the readme.md. But when I trying to enter this repo it always show me 404 : https://github.com/Jiazi345/Jiazi345.github.io

![屏幕截图 2022-10-31 202729](https://user-images.githubusercontent.com/97338032/199007750-e83dfe41-8b50-4bd9-9b7b-93c32301b992.jpg)

but if I input https://jiazi345.github.io/ it shows a initial page:

![屏幕截图 2022-10-31 200231](https://user-images.githubusercontent.com/97338032/199007189-64ece69a-2c72-4dca-bcbd-80682b63420d.jpg)

However I can't enter this  repo so I don't know how to change any setting or push documents or delect it.  and I try to clone this repo in local but failed ↓  what should I do ?  how can I come to my reposity?
![屏幕截图 2022-10-31 203530](https://user-images.githubusercontent.com/97338032/199009203-f55c5234-53e3-4939-9d35-47d1438a79ed.jpg)

**Comment by ghost** [comment]
When I visit [https://github.com/Jiazi345/Jiazi345.github.io](https://github.com/Jiazi345/Jiazi345.github.io):

![image](https://user-images.githubusercontent.com/50993163/199012482-40c3c1f0-fa8b-46fd-bc01-213304c199d2.png)

When I visit [https://jiazi345.github.io](https://jiazi345.github.io):

![image](https://user-images.githubusercontent.com/50993163/199012421-5ca52352-c02c-4657-9ccc-1079d9152be9.png)

**Comment by jfullstackdev** [comment, marked as answer]
you might want to try again, I don't have any issues accessing both, 

try logging in in Incognito mode too

**Reply by Jiazi345** [reply]
Thank you!!It works when I changed my network.

**Reply by Talalgmel** [reply]
Yy
